modules/network/src/integrationTest/java/org/apache/ignite/internal/network/netty/ItConnectionManagerTest.java

```diff
@@ -296,6 +300,61 @@ public void testStopTwice() throws Exception {
         server.close();
     }
 
+    @Test
+    public void testOneConnectionType() throws Exception {
+        int size = 10000000;
+        char[] chars = new char[size];
+        Arrays.fill(chars, 'a');
+        String bigText = new String(chars);
+
+
+        ConnectionManagerWrapper server1 = startManager(4000);
+        ConnectionManagerWrapper server2 = startManager(4001);
+
+        NettySender sender = server1.openChannelTo(server2, ChannelType.DEFAULT).get(3, TimeUnit.SECONDS);
+
+
+        TestMessage bigMessage = messageFactory.testMessage().msg(bigText).build();
+        TestMessage msg = messageFactory.testMessage().msg("test").build();
+
+        CompletableFuture<Void> send1 = sender.send(new OutNetworkObject(bigMessage, Collections.emptyList()));
+        CompletableFuture<Void> send2 = sender.send(new OutNetworkObject(msg, Collections.emptyList()));
+
+        assertThat(send2, willCompleteSuccessfully());
+        assertThat(send1.isDone(), is(true));
```

Comment:
I understood what you are testing here but it's not obvious. Could you please change the naming like: `firstSend`, `secondSend`. Because order matters here. 

Also, `bigMessage` is ok but why there is no `smallMassage`?